pkg/repository/v1manifest/local_manifests.go

```diff
@@ -143,6 +143,12 @@ func (ms *FsManifests) LoadManifest(role ValidManifest) (*Manifest, bool, error)
 
 	m, err := ReadManifest(strings.NewReader(manifest), role, ms.keys)
 	if err != nil {
+		// We can think that there is no such file if it's expired or has wrong signature
+		if IsExpirationError(errors.Cause(err)) || IsSignatureError(errors.Cause(err)) {
+			// Maybe we can os.Remove(filename) here
+			fmt.Printf("Warn: %s\n", err.Error())
```

Comment:
I don't think we should print any warning here because expiration is expected.

Reply:
But the signature expectation is not.
Maybe only warn if it's signature exception?

Reply:
Yes, SGTM